config/logging.go

```diff
@@ -17,6 +17,7 @@ const (
 type LoggerConfig struct {
 	Encoder                   LogEncoder `mapstructure:"log-encoder"`
 	AppLoggerLevel            string     `mapstructure:"app"`
+	GrpcLoggerLevel           string     `mapstructure:"grpc"`
```

Comment:
This field needs a default value (a few lines down in this file)

Reply:
Interesting that it works anyway and gets an INFO level by default 🤷